guides/micronaut-static-resources/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+micronaut.application.name=staticresources
+# tag::assets[]
+micronaut.router.static-resources.assets.mapping=/**
```

Comment:
Instead of using such an open wildcard. Use something like 

```
micronaut.router.static-resources.css.mapping=/css/*.css
micronaut.router.static-resources.css.paths=classpath\:static/css
micronaut.router.static-resources.images.mapping=/images/*.svg
micronaut.router.static-resources.images.paths=classpath\:static/images
```

Reply:
Unsure what this gives us, but if we do this, can we do:

```
micronaut.router.static-resources.images.mapping=/images/**
```

instead?

Reply:
With such setting: 

`micronaut.router.static-resources.assets.mapping=/**`

The framework searches for every request a controller route; if not found, it searches for a static resource. 

Searching for a static resource in every request is a waste. We know specific paths (e.g., /bogus) will not match any resource. 

We only want to search for static resources for paths starting with `/css` or `/images`. 

> can we do:

> micronaut.router.static-resources.images.mapping=/images/**
> instead?

Yes, we can do `micronaut.router.static-resources.images.mapping=/images/**` instead of `micronaut.router.static-resources.images.mapping=/images/*.svg` if you prefer.

Reply:
Done in https://github.com/micronaut-projects/micronaut-guides/pull/1426/commits/b14d90a489871a33b096daae322005ee3bd8576b with an extra section added to the guide to explain why it is better this way (stolen from your comment above 😉)